lib/awful/ewmh.lua

```diff
@@ -133,7 +137,19 @@ end
 --        if c.class == "Firefox" then return false end
 --    end, "ewmh")
 --
+-- Or to activate the first tag of not visible clients (to not make the default
+-- handler mark them urgent only):
+--
+--    awful.ewmh.add_activate_filter(function(c, _context, hints)
+--        if hints.raise and not c:isvisible() then
+--            c.first_tag:view_only()
+--        end
+--    end, "ewmh")
```

Comment:
Does this example make sense like this?
This does not return anything on purpose, but in theory another callback might return `false` later.

Could `hints` be used for this, and then this filter would enforce `hints.make_visibible`?

Reply:
Umm, that's an interesting question. I don't think this is the right place to do this kind of things, but I found no better place either.

What do you think about emitting a `require::tag {raise=true}` (or `request::activate`,  from both `awful.rules.delayed_properties.switchtotag` and `awful.ewmh.activate` and extending `awful.ewmh.tag` to handle raising tags? This would be more consistent with the intend behind your idea considering how it's done for other assets.

Reply:
Ok, will likely remove / move it to a new issue then - not now though.

Reply:
v4.3 is approaching, the issues tagged with the v4.3 milestone are about to be fixed. If you want this and some of your other PRs to be part of the release, please update them.